client/src/App.jsx

```diff
@@ -1,34 +1,119 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from 'react'
+import { Flex, Container, Box, Card} from '@radix-ui/themes';
+import { Heading, Text, TextField, Button, SegmentedControl, Avatar, ScrollArea, Select, Link } from '@radix-ui/themes';
+import { AccessibilityIcon } from '@radix-ui/react-icons'
+import { makeServer } from './server'
+
+if (process.env.NODE_ENV === 'development') {
+  makeServer()
+}
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [inputValue, setInputValue] = useState('');
+  const [responseData, setResponseData] = useState(null);
+
+  // Get threads for first AI msg
+  useEffect(() => {
+    handleLoad()
```

Comment:
Always remember to catch the error for `async` function.
```suggestion
    handleLoad().catch(console.error);
```